go.mod

```diff
@@ -5,12 +5,13 @@ go 1.20
 require (
 	github.com/go-logr/logr v1.2.4
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.66.0
```

Comment:
should we use a more recent version?

Reply:
It is safer to follow what CMO has https://github.com/openshift/cluster-monitoring-operator/blob/master/go.mod#L16

Reply:
I did look into any API changes to ServiceMonitor from 0.66, didn't find any... however, didn't want to risk it either :D